eks_addon/release.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+# Builds, packages, and pushes artifacts for the Splunk OpenTelemetry Collector EKS Add-on.
+#
+# Parameters:
+#   --dry-run : (Optional) Enables dry-run mode that skips pushing artifacts.
+#   --chart-version VERSION : (Optional) Overrides the chart version from Chart.yaml.
+
+# Enable bash strict mode to fail fast
+set -euo pipefail
+
+# Check required tools
+for tool in "okta-aws-login" "aws" "yq" "helm" "docker"; do
+  command -v "${tool}" &>/dev/null || { echo "❌ Required command '${tool}' is not installed or not in PATH"; exit 1; }
+done
+
+# Parse command line arguments
+DRY_RUN_PREFIX=""
+CHART_VERSION=""
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --dry-run)
+      DRY_RUN_PREFIX="echo 🚧 [DRY-RUN] "
+      shift
+      ;;
+    --chart-version)
+      CHART_VERSION="$2"
+      echo "ℹ️ Chart version override provided: $CHART_VERSION"
+      shift 2
+      ;;
+    *)
+      echo "❌ Unknown argument: $1"
+      exit 1
+      ;;
+  esac
+done
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CHART_DIR="$SCRIPT_DIR/../helm-charts/splunk-otel-collector"
+if [[ -z "$CHART_VERSION" ]]; then
+  CHART_VERSION=$(yq e ".version" "${CHART_DIR}/Chart.yaml")
+fi
+CHART_APPVERSION=$(yq e ".appVersion" "${CHART_DIR}/Chart.yaml")
+ECR_REGISTRY="709825985650.dkr.ecr.us-east-1.amazonaws.com"
+ECR_REGION="us-east-1"
+ECR_OTELCOL_REPO="${ECR_REGISTRY}/splunk/images/splunk-otel-collector"
+ECR_FLUENTD_REPO="${ECR_REGISTRY}/splunk/docker.io/splunk/fluentd-hec"
+ECR_FLUENTD_REPO_TAG="1.3.3-linux"
+ECR_HELM_NAMESPACE="${ECR_REGISTRY}/splunk/charts"
+ECR_HELM_REPO="${ECR_HELM_NAMESPACE}/splunk-otel-collector"
+
+aws_okta_auth() {
+  echo "🌐 Authenticating with Okta (browser will open)..."
+  okta-aws-login > /tmp/okta_creds.txt 2>&1 || { echo "❌ AWS Authentication failed"; rm -f /tmp/okta_creds.txt; exit 1; }
```

Comment:
This command is hanging for me, even after the browser says I've successfully authenticated:
```
$ ./eks_addon/release.sh 
🌐 Authenticating with Okta (browser will open)...
```
Any debugging information or comments we could add to clarify usage?

Reply:
Yeah. It's expected. It should be run with `OKTA_AWS_ROLE_ARN` env var set. It'll be part of the release doc